README.rdoc

```diff
@@ -6,6 +6,39 @@
 
 A simpler way to builder -kit for building Java packages
 
+== Generating source packages
+
+The `jarcheck` utility can generate source packages from a
+[Tetra](https://github.com/moio/tetra) kit tarball. To use it, proceed as
+follows:
+
+1. Change to the future source package's directory.
+
+2. Create a directory to unpack the kit tarball in. This directory must be
+   named `<name>-kit`, where `<name>` is the main package's name. Example: if
+   your main package is named `foo`, this directory must be named `foo-kit`.
+
```

Comment:
Do we need to include a warning about : in the pathnames? Is that a problem as it is for tetra?

Reply:
The tool shouldn't enforce a specific directory name imho.